test/server/cards/01_SOR/DirectorKrennicAspiringToAuthority.spec.js

```diff
@@ -0,0 +1,80 @@
+describe('Director Krennic, Aspiring to Authority', function() {
+    integration(function() {
+        describe('Krennic\'s undeployed ability', function() {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: [{ card: 'wampa', damage: 1 }, 'battlefield-marine'],
+                        spaceArena: [{ card: 'cartel-spacer', damage: 1 }],
+                        leader: 'director-krennic#aspiring-to-authority'
+                    },
+                    player2: {
+                        groundArena: [{ card: 'consular-security-force', damage: 1 }],
+                    }
+                });
+            });
+
+            it('should give friendly damaged units +1/+0', function () {
+                expect(this.wampa.power).toBe(5);
+                expect(this.wampa.hp).toBe(5);
+
+                expect(this.battlefieldMarine.power).toBe(3);
+                expect(this.battlefieldMarine.hp).toBe(3);
+
+                expect(this.cartelSpacer.power).toBe(3);
+                expect(this.cartelSpacer.hp).toBe(3);
+
+                expect(this.consularSecurityForce.power).toBe(3);
+                expect(this.consularSecurityForce.hp).toBe(7);
```

Comment:
shouldnt this be 6? theres 1 damage on it in setup

Reply:
I followed the game rules pretty literally so HP always refers to max HP value, damage is not being subtracted out. The unit is defeated when damage == HP

Reply:
I think this is a little unintuitive so I might suggest we rename the hp to maxHp if that's the case

Reply:
That’s reasonable, I could go either way. I’ll poll in the chat to see what people find clearest.